core/src/server/web_impl/Constants.h

```diff
@@ -52,11 +52,11 @@ static const char* NAME_METRIC_TYPE_SUPERSTRUCTURE = "SUPERSTRUCTURE";
 
 ////////////////////////////////////////////////////
 
-static const char* KEY_TABLE_TABLE_NAME = "table_name";
-static const char* KEY_TABLE_DIMENSION = "dimension";
-static const char* KEY_TABLE_INDEX_FILE_SIZE = "index_file_size";
-static const char* KEY_TABLE_INDEX_METRIC_TYPE = "metric_type";
-static const char* KEY_TABLE_COUNT = "count";
+static const char* KEY_COLLECTION_TABLE_NAME = "table_name";
```

Comment:
stop using 'static const char*' in header files. Use .cpp to have these variables.

Reply:
solved.